web/src/SidebarResources.test.tsx

```diff
@@ -132,8 +135,108 @@ describe("SidebarResources", () => {
     expectIncr(0, "ui.web.pin", { pinCount: "2", action: "load" })
     expectIncr(1, "ui.web.pin", { pinCount: "1", action: "unpin" })
 
-    expect(localStorage.getItem(makeKey("test", "pinned-resources"))).toEqual(
-      JSON.stringify(["vigoda"])
-    )
+    expect(pinnedItemsAccessor.get()).toEqual(["vigoda"])
   })
+
+  const loadCases: [string, SidebarOptions, string[]][] = [
+    [
+      "showResources",
+      { showResources: false, showTests: true, alertsOnTop: true },
```

Comment:
I believe the default value of `alertsOnTop` is false -- not a big thing but if you're trying to test one property at a time getting loaded from localStorage and being different from the default, you should flip all the `alertsOnTop` values.